frontend/fingertips-frontend/upload/components/pages/upload/index.tsx

```diff
@@ -17,11 +20,16 @@ type UploadProps = {
 };
 
 export const Upload = ({ batches }: Readonly<UploadProps>) => {
+  const [deleteResponse, setDeleteResponse] = useState(
+    null as unknown as ApiResponse
+  );
```

Comment:
You can pass a type to `useState` when initialising it with an empty value, rather than having to do `null as unknown as ApiResponse`:
```suggestion
  const [deleteResponse, setDeleteResponse] = useState<ApiResponse | null>(
    null
  );
```